src/jit/codegencommon.cpp

```diff
@@ -3766,31 +3766,52 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
         return;
     }
 #endif
+    unsigned    argMax;            // maximum argNum value plus 1, (including the RetBuffArg)
```

Comment:
This is easy to miss - it might be nice to have a comment somewhere more visible that explains that we have an additional arg entry for the RetBuffArg, if present. And, does argMax always include a potential RetBuffArg even when not present (which this comment seems to imply)?